reference/docs-conceptual/windows-powershell/install/Windows-PowerShell-System-Requirements.md

```diff
@@ -20,85 +20,64 @@ users of earlier releases of Windows.
-| Windows Server 2022                        | Higher version installed by default                                     |
-| Windows Server 2019                        | Higher version installed by default                                     |
-| Windows Server 2016                        | Higher version installed by default                                     |
-| Windows Server 2012 R2                     | Install [Windows Management Framework 5.1](https://aka.ms/wmf5download) |
-| Windows Server 2012                        | Install [Windows Management Framework 5.1](https://aka.ms/wmf5download) |
-| Windows Server 2008 R2 with Service Pack 1 | Install [Windows Management Framework 5.1](https://aka.ms/wmf5download) |
-| Windows 11                                 | Higher version installed by default                                     |
-| Windows 10 version 1607 and up             | Higher version installed by default                                     |
-| Windows 10 version 1507, 1511              | Installed by default                                                    |
-| Windows 8.1                                | Install [Windows Management Framework 5.1](https://aka.ms/wmf5download) |
-| Windows 7 with Service Pack 1              | Install [Windows Management Framework 5.1](https://aka.ms/wmf5download) |
+Windows Management Framework 5.1 supersedes Windows Management Framework 5.0. For more information,
+see [Install and Configure WMF 5.1][04].
+
+|              Windows version               |               System requirement               |
+| ------------------------------------------ | ---------------------------------------------- |
+| Windows Server 2022                        | Higher version installed by default            |
+| Windows Server 2019                        | Higher version installed by default            |
+| Windows Server 2016                        | Higher version installed by default            |
+| Windows Server 2012 R2                     | Install [Windows Management Framework 5.1][18] |
+| Windows Server 2012                        | Install [Windows Management Framework 5.1][18] |
+| Windows Server 2008 R2 with Service Pack 1 | Install [Windows Management Framework 5.1][18] |
+| Windows 11                                 | Higher version installed by default            |
+| Windows 10 version 1607 and up             | Higher version installed by default            |
+| Windows 10 version 1507, 1511              | Installed by default                           |
+| Windows 8.1                                | Install [Windows Management Framework 5.1][18] |
+| Windows 7 with Service Pack 1              | Install [Windows Management Framework 5.1][18] |
 
 ### Windows PowerShell 4.0
 
 Windows PowerShell 4.0 runs on the following versions of Windows. To run Windows PowerShell 4.0,
 install the specified version of the Windows Management Framework for your operating system.
```

Comment:
Should we strike the install language here? Possibly mention that people should install 5.1 instead?
```suggestion
Windows PowerShell 4.0 runs on the following versions of Windows.
```